man/GGIR.Rd

```diff
@@ -686,6 +692,17 @@ GGIR(mode = \Sexpr{format(formals(GGIR::GGIR)[["mode"]])},
         and acceleration metrics to be imputed by zeros. The data.frame is expected
         to contain two columns named windowstart and windowend, with the start- and end
         time of the time segment in POSIXlt class.}
+        
+      \item{part2ImpWithZero}{
```

Comment:
It may be good to motivate here in what scenarios this argument is relevant. It sounds like this is only relevant for studies where accelerometer was not worn during the night?

Reply:
Done